pages/common/john.md

```diff
@@ -25,12 +26,12 @@
 
 - Crack password hashes, using a specific hash format:
 
-`john --format={{md5crypt|...}} {{path/to/hashes.txt}}`
+`john --format={{hash_format}} {{path/to/hashes.txt}}`
```

Comment:
What's the reason behind this change?

Reply:
I actually just wanted to remove the `...` and not change the concrete example. My reasoning was that we use the `|` format to show alternative options, but only when they are complete. If all options cannot be specified, then it's simple to just use one option, as it's implicit that it can be replaced by other options.

Reply:
Why not just put `md5crypt` here instead?

Reply:
That's what I am suggesting to do here :) Perhaps the way I put it across might have meant something else.

Reply:
Ah, I see. I must have misinterpreted. I'm somewhat cautious of including `md5crypt` on it's own here, sincce md5 is known to be a weak hashing algorithm - and this is a password-based tool. But perhaps it will be ok because this is a password cracking tool, rather than a hashing.